### DISCUSSION TITLE
How to install OpenSSL on Windows machine

### DISCUSSION THREAD

**Discussion by NetworkMonkey101** [openssl/openssl#25505] [General Discussion]
Hi, first time poster here. 
I am currently working with Cisco FTD and am trying to fix a CA cert within the device. I have been shared this link - https://www.linkedin.com/pulse/anyconnect-ftd-pkcs12-openssl-matt-albrecht/

After downloading the tar file from - https://openssl-library.org/source/index.html how do I install it on my Windows machine to follow the steps in the above linkedin guide?

**Comment by nhorman** [comment]
You have to build it.  The tar file is similar to a zip file.  Use 7z or something like WinZip to extract the files from the tarball, then consult the NOTES-WINDOWS.md file for build instructions
